with-coingecko/index.graphql

```diff
@@ -0,0 +1,21 @@
+schema
+  @sdl(
+    files: [
+      "types/assets.graphql"
```

Comment:
typically we put all the generated `.graphql` files in a folder named after the data source. So `types` would be renamed to `coingecko`. I'll add this to the contributing guide